### DISCUSSION TITLE
ENV Setting MINIO_BROWSER_REDIRECT_URL Broken?

### DISCUSSION THREAD

**Discussion by RR-Helpdesk** [minio/minio#17897] [Q&A] [answered]
ENV Setting MINIO_BROWSER_REDIRECT_URL & MINIO_SERVER_URL Causing Error

## Expected Behavior
Should function as explained in the docs.

## Current Behavior

DOCS INSTRUCTIONS:
For example, consider a MinIO deployment behind a proxy https://minio.example.net, https://console.minio.example.net with rules for forwarding traffic on port :9000 and :9001 to MinIO and the MinIO Console respectively on the internal network. Set MINIO_BROWSER_REDIRECT_URL to https://console.minio.example.net to ensure the browser receives a valid reachable URL.

WHEN ADDING THE PARAMETERS:
MINIO_BROWSER_REDIRECT_URL="https://minio.docker.XXXX.cloud/console"
MINIO_SERVER_URL="https://minio.docker.XXXX.cloud"

ERROR:

The following error then occurs on the docker container preventing start:

ERROR Invalid MINIO_BROWSER_REDIRECT_URL value in environment variable: parse "\"https://minio.docker.XXXX.cloud/console\"": first path segment in URL cannot contain colon

ERROR Invalid MINIO_SERVER_URL value in environment variable: parse "\"https://minio.docker.revrebel.cloud\"": first path segment in URL cannot contain colon

## Your Environment

* Version used (`minio --version`):  minio/minio:latest
* Server setup and configuration: Linux/Docker
* Operating System and version (`uname -a`): Linux docker.XXX.cloud 5.15.0-78-generic #85-Ubuntu SMP Fri Jul 7 15:25:09 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

  minio-server:
    image: minio/minio:latest
    hostname: minio
    container_name: minio
    expose:
      - "9000"
      - "9001"
    networks:
      - docweb
    environment:
      - TZ=America/Los_Angeles
      - PUID=1000
      - PGID=1000
      - MINIO_ROOT_USER="XXXX"
      - MINIO_ROOT_PASSWORD="XXXX"
      - MINIO_API_ROOT_ACCESS="on"
      - MINIO_DOMAIN="minio.docker.XXXX.cloud"
      - MINIO_BROWSER_REDIRECT_URL="https://minio.docker.XXXX.cloud/console"
      - MINIO_SERVER_URL="https://minio.docker.XXXX.cloud"
   volumes:
      - ./storage:/data
    command: server --address 0.0.0.0:9000 --console-address 0.0.0.0:9001 /data
    extra_hosts:
      - "host.docker.internal:host-gateway"

**Comment by jiuker** [comment, marked as answer]
MINIO_BROWSER_REDIRECT_URL=https://minio.docker.xxxx.cloud/console
MINIO_SERVER_URL=https://minio.docker.xxxx.cloud/   @RR-Helpdesk try it. Delete `"`
